languages/csharp/exercises/concept/enums/.meta/Example.cs

```diff
@@ -1,4 +1,4 @@
-public enum LogLevel
```

Comment:
Should ln#12 be changed also?

Reply:
Doh. I forgot to update the `public static` classes too. Update the PR.